test/build/50_ch-tug.bats

```diff
@@ -234,3 +234,48 @@ EOF
     [[ $output = 'regular file' ]]
     [[ $(cat "${img}/test/source") = 'regular' ]]
 }
+
+@test 'ch-tug image with specified schema' {
+    # Validate that we handle both schema version (1) and (2) image formats.
+    scope standard
+    if ( ! ch-tug --dependencies ); then
+
+        [[ $CH_BUILDER != ch-grow ]]
+        skip "ch-tug missing dependencies"
+    fi
+
+    unpack=$BATS_TMPDIR
+    cache=$unpack/dlcache
+    img=charliecloud%symlink
```

Comment:
Why this image?

Reply:
No specific reason. Do you prefer another?

Reply:
Nothing specific in mind, but that image was created for another purpose, so we should justify its use here.